cli/README.md

```diff
@@ -48,17 +48,17 @@ mkdir deploy
 Build and copy Iroha binary into the directory. 
 
 ```bash
-cargo build
-cp ./target/debug/iroha deploy
+cargo build --release
+cp ./target/release/iroha deploy
 ```
 
 #### Copy configs
 
 Copy and if necessary edit config, genesis and trusted peers.
 ```bash
-cp ./core/config.json deploy
-cp ./core/genesis.json deploy
-cp ./core/trusted_peers.json deploy
+cp ./configs/docker/config.json deploy
+cp ./configs/docker/genesis.json deploy
+cp ./configs/docker/trusted_peers.json deploy
```

Comment:
I like `peer` more. Not least because the contents of the `configs` folder are `peer` and `client_cli` which are exactly the two use-cases for our library: starting a peer and connecting to a network.